index.html

```diff
@@ -6,37 +6,50 @@
     <title>Balázs házi feladata</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
-    <nav>
-        <p>Ez itt a navigáció</p>
-        <a href=#alja>Ugrás a lap aljára</a>
+<body class="theme-a adaptive">
+    <nav id="teteje" class="theme-a">
+        <ul>
+            <li><a href="#teteje">Bevezetés</a></li>
+            <li><a href="#kozepe">Tartalom</a></li>
+            <li><a href="#alja">Vége</a></li>
+            <li><a href="extra.html" target="_blank">További információ</a></li>
+            <a class="center" href=#alja>Ugrás a lap aljára</a>
```

Comment:
Sajnos ettől a sortól invalid lett a html.

Reply:
Attribútum értékeket zárd be idézőjelek közé.

Reply:
Kijavítottam. Egyébként a href=teteje átmegy hiba nélkül a validátoron.